db/pre-re/status.yml

```diff
@@ -3112,11 +3106,12 @@ Body:
       StopAttacking: true
       OverlapFail: true
     End:
-      Dancing: true
       Burning: true
       Freezing: true
       Freeze: true
       Stone: true
+      StoneWait: true
+      Sleep: true
```

Comment:
I didn't know Whiteimprison is on pre-re, but since it is, shouldn't we also add it to the fail/end of other OPT1 skills? (This is mainly to make it easier for people who want to use this in pre-re.)
It's also still missing "Stun" on fail.